config/samples/certificate.yaml

```diff
@@ -1,40 +0,0 @@
----
-apiVersion: cert-manager.io/v1alpha2
```

Comment:
Needs updating with the content of the README. So can be "reverted" with those new changes